packages/babel-plugin-proposal-object-rest-spread/src/index.ts

```diff
@@ -1,21 +1,25 @@
 import { declare } from "@babel/helper-plugin-utils";
 import syntaxObjectRestSpread from "@babel/plugin-syntax-object-rest-spread";
 import { types as t } from "@babel/core";
+import type { PluginPass } from "@babel/core";
+import type { NodePath, Visitor, Scope } from "@babel/traverse";
 import { convertFunctionParams } from "@babel/plugin-transform-parameters";
 import { isRequired } from "@babel/helper-compilation-targets";
 import compatData from "@babel/compat-data/corejs2-built-ins";
 import shouldStoreRHSInTemporaryVariable from "./shouldStoreRHSInTemporaryVariable";
 
-// TODO: Remove in Babel 8
+const { isAssignmentPattern, isObjectProperty } = t;
 // @babel/types <=7.3.3 counts FOO as referenced in var { x: FOO }.
 // We need to detect this bug to know if "unused" means 0 or 1 references.
-const ZERO_REFS = (() => {
-  const node = t.identifier("a");
-  const property = t.objectProperty(t.identifier("key"), node);
-  const pattern = t.objectPattern([property]);
+const ZERO_REFS = process.env.BABEL_8_BREAKING
```

Comment:
I'd prefer to do
```js
if (!process.env.BABEL_8_BREAKING) {
  // eslint-disable-nenxt-line prefer-const
  var ZERO_REFS = ...
}
```
and then replace `ZERO_REFS` usages with `process.env.BABEL_8_BREAKING ? 0 : ZERO_REFS`, so that when codemod'ing `process.env.BABEL_8_BREAKING` away `ZERO_REFS` disappears.